src/Illuminate/Auth/Console/stubs/make/tests/AuthTest.stub

```diff
@@ -0,0 +1,62 @@
+<?php
+
+use {{namespace}}User;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+
+class AuthTest extends TestCase
+{
+    use DatabaseMigrations;
+
+    protected $name = 'Your name';
+    protected $email = 'a@valid.email';
+    protected $password = 'secret';
+
+    public function test_registration()
+    {
+        $this->visit('register')
+            ->type($this->name, 'name')
+            ->type($this->email, 'email')
+            ->type($this->password, 'password')
+            ->type($this->password, 'password_confirmation')
+            ->press('Register')
+            ->seeInDatabase('users', [
+                'name'  => $this->name,
+                'email' => $this->email,
+            ])
+            ->see('Welcome')
+            ->seePageIs('/')
+            ->seeIsAuthenticated();
```

Comment:
https://github.com/laravel/framework/pull/12533

I added a new method for the integrated test package called "seeCredentials". So instead of:

```
->seeInDatabase('users', [
    'name'  => $this->name,
    'email' => $this->email,
])
```

We could use: 

```
->seeCredentials([
    'name'  => $this->name,
    'email' => $this->email,
    'password' => $this->password,
])
```

That'd check not only that the user was saved, but that the password was correctly encrypted and saved as well. 

So if that commit is merged, then I'll modify this PR to use seeCredentials instead of seeInDatabase :)